app/src/App.js

```diff
@@ -4,6 +4,7 @@ import 'bootstrap/dist/js/bootstrap';
 import { useContext } from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import { GameContext } from './contexts/GameContext';
+import { HandContext } from './contexts/HandContext';
```

Comment:
Szerintem nem különösebben indokolt egy külön HandContext, mehetne az egész a GameContextbe is